includes/admin/class-wp-job-manager-admin.php

```diff
@@ -130,7 +130,7 @@ public function admin_enqueue_scripts() {
 			);
 		}
 
-		WP_Job_manager::register_script( 'job_manager_notice_dismiss', 'js/admin/wpjm-notice-dismiss.js', [], true );
+		WP_Job_manager::register_script( 'job_manager_notice_dismiss', 'js/admin/wpjm-notice-dismiss.js', null, true );
```

Comment:
The `WP_Job_manager::register_script` helper will load the dependencies from `[name].assets.php` generated the webpack script only if this argument is null. Supplying an empty array will instead override that dependency list.